identity-service/default-config.json

```diff
@@ -66,6 +66,5 @@
   "hCaptchaSecret": "",
   "solanaFeePayerWallet": "[]",
   "sentryDSN": "",
-  "solanaEndpoint": "https://api.mainnet-beta.solana.com",
-  "optimizelySdkKey": ""
+  "solanaEndpoint": "https://api.mainnet-beta.solana.com"
```

Comment:
why remove the optimizely config?

Reply:
I added the staging key as the default in config.js, and then it was getting overwritten by this empty string here. Seems weird we have defaults in two places, so I opted for config.js. Objections?

Reply:
ahhh no this makes sense!